README.md

```diff
@@ -55,7 +55,7 @@ This will tear down infrastructure provisioned by `provisionInfrastructure` and
 
 ### provisionPrDatabaseRoleAndSchema
 
-Creates a Postgres database role and schema for use by a PR.
+Creates a Postgres database role and schema for use by a PR. See https://eaflood.atlassian.net/wiki/spaces/FPS/pages/1596653973/Creating+PR+database+namespaces+in+postgres for context.
 
```

Comment:
You could you use the `[link-url](link description)` format to make the url look neater